Makefile

```diff
@@ -44,19 +44,18 @@ lint:
 	make lint.run
 
 lint.install:
-	python -m pip install --upgrade pip
 	echo "Installing..."
-	pip install -Iv black==22.3.0 isort
+	pip install -Iv ruff
 
```

Comment:
Is this file the only non auto generated change in the PR?

Reply:
and the ruff.toml which has some slight tweaks from the default here: https://docs.astral.sh/ruff/configuration/

Reply:
Are the rules same as the the worker repo?

Reply:
`worker` is more or less using the default except for the extra linting of `F401`. It's good that you added the `ruff.toml` file!